src/types/account-id.ts

```diff
@@ -56,6 +64,8 @@ class AccountID extends Hash160 {
    * @returns the base58 string defined by this.bytes
    */
   toBase58(): string {
+    if (this.bytes.byteLength === 0) return "";
```

Comment:
If this is a change, can we add a test to prove the correctness?

Reply:
Yes